server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/Node.java

```diff
@@ -35,22 +35,14 @@
 public abstract class Node {
   private final Locality locality;
   private final NodeType nodeType;
-  private final boolean oneRun;
 
   @EqualsAndHashCode.Exclude private final List<Node> children = new ArrayList<>();
   @EqualsAndHashCode.Exclude @ToString.Exclude @Setter private Node parent;
-  @EqualsAndHashCode.Exclude private boolean processed = false;
+  @EqualsAndHashCode.Exclude private Optional<Supplier<List<SyntaxError>>> processIt = Optional.empty();
```

Comment:
I would pick a more descriptive name

Reply:
`processHandler` ?

Reply:
`processingHandler` maybe? It should be understandable for one what didn't work on the Tree.

Reply:
Or maybe `nextProcessingStep`, just because it can be set via `addProcessStep`.

Reply:
Makes sense